### DISCUSSION TITLE
Why Firefox?

### DISCUSSION THREAD

**Discussion by leonidcan** [FlareSolverr/FlareSolverr#222] [Q&A]
Please explain why switch to Firefox? Post method is completely broken now. :(

**Comment by ngosang** [comment]
The main goal of this project is to bypass Cloudflare protection. After the latest changes in Cloudflare challenge, Chrome+Puppeter is always detected even if you use stealth plugins.
It's true that the POST method is not working well but at least the GET method is working. You can make a GET request to grab the cookies and then write your own script to make the POST request with the Cloudflare headers + FlareSolverr User-Agent.

**Reply by leonidcan** [reply]
Interesting... In my scenario it's still fine to use Chrome+Puppeter  (with stealth plugin). So for the get/post workaround does it mean I need to keep cookies between calls? I am trying to be prepared if there will be another change. I mean I have to call get once per session and then just reuse cookies for subsequent posts?

**Reply by ngosang** [reply]
You can use FlareSolverr 1.x.x or even fork the project if that fix your issue.

**Reply by DRSDavidSoft** [reply]
@ngosang Just wondering, will you be willing to accept PRs into the 1.x.x branch? In order to back port useful changes?

**Reply by ngosang** [reply]
Sure, if you are able to bypass Cloudflare with Google Chrome I can even publish releases.

**Reply by DRSDavidSoft** [reply]
@ngosang That would be awesome, I'm currently using v1.2.5 without any issues!  Even with the missing updates, FlareSolverr is solving Cloudflare with ease.
